libs/product/src/drivers/magento/models/bundled-product.ts

```diff
@@ -13,10 +13,6 @@ export enum MagentoPriceTypeEnum {
 }
 
 export interface MagentoBundledProduct extends MagentoProduct {
```

Comment:
Let's remove this from this PR, its outside scope.

Reply:
reverted